src/languages/sqf.js

```diff
@@ -1,2529 +1,2630 @@
+    'units',
+    'unitsBelowHeight',
+    'unitTurret',
+    'unlinkItem',
+    'unlockAchievement',
+    'unregisterTask',
+    'updateDrawIcon',
+    'updateMenuItem',
+    'updateObjectTree',
+    'useAIOperMapObstructionTest',
+    'useAISteeringComponent',
+    'useAudioTimeForMoves',
+    'userInputDisabled',
+    'values',
+    'vectorAdd',
+    'vectorCos',
+    'vectorCrossProduct',
+    'vectorDiff',
+    'vectorDir',
+    'vectorDirVisual',
+    'vectorDistance',
+    'vectorDistanceSqr',
+    'vectorDotProduct',
+    'vectorFromTo',
+    'vectorLinearConversion',
+    'vectorMagnitude',
+    'vectorMagnitudeSqr',
+    'vectorModelToWorld',
+    'vectorModelToWorldVisual',
+    'vectorMultiply',
+    'vectorNormalized',
+    'vectorUp',
+    'vectorUpVisual',
+    'vectorWorldToModel',
+    'vectorWorldToModelVisual',
+    'vehicle',
+    'vehicleCargoEnabled',
+    'vehicleChat',
+    'vehicleMoveInfo',
+    'vehicleRadio',
+    'vehicleReceiveRemoteTargets',
+    'vehicleReportOwnPosition',
+    'vehicleReportRemoteTargets',
+    'vehicles',
+    'vehicleVarName',
+    'velocity',
+    'velocityModelSpace',
+    'verifySignature',
+    'vest',
+    'vestContainer',
+    'vestItems',
+    'vestMagazines',
+    'viewDistance',
+    'visibleCompass',
+    'visibleGps',
+    'visibleMap',
+    'visiblePosition',
+    'visiblePositionASL',
+    'visibleScoretable',
+    'visibleWatch',
+    'waves',
+    'waypointAttachedObject',
+    'waypointAttachedVehicle',
+    'waypointAttachObject',
+    'waypointAttachVehicle',
+    'waypointBehaviour',
+    'waypointCombatMode',
+    'waypointCompletionRadius',
+    'waypointDescription',
+    'waypointForceBehaviour',
+    'waypointFormation',
+    'waypointHousePosition',
+    'waypointLoiterAltitude',
+    'waypointLoiterRadius',
+    'waypointLoiterType',
+    'waypointName',
+    'waypointPosition',
+    'waypoints',
+    'waypointScript',
+    'waypointsEnabledUAV',
+    'waypointShow',
+    'waypointSpeed',
+    'waypointStatements',
+    'waypointTimeout',
+    'waypointTimeoutCurrent',
+    'waypointType',
+    'waypointVisible',
+    'weaponAccessories',
+    'weaponAccessoriesCargo',
+    'weaponCargo',
+    'weaponDirection',
+    'weaponInertia',
+    'weaponLowered',
+    'weaponReloadingTime',
+    'weapons',
+    'weaponsInfo',
+    'weaponsItems',
+    'weaponsItemsCargo',
+    'weaponState',
+    'weaponsTurret',
+    'weightRTD',
+    'WFSideText',
+    'wind',
+    'windDir',
+    'windRTD',
+    'windStr',
+    'wingsForcesRTD',
+    'worldName',
+    'worldSize',
+    'worldToModel',
+    'worldToModelVisual',
+    'worldToScreen'
+    ];
+
+  // list of keywords from:
+  // https://community.bistudio.com/wiki/PreProcessor_Commands
+  const PREPROCESSOR = {
+    className: 'meta',
+    begin: /#\s*[a-z]+\b/,
+    end: /$/,
+    keywords: {
+      keyword:
+        'define undef ifdef ifndef else endif include'
+    },
+    contains: [
+      {
+        begin: /\\\n/,

```

Comment:
For line continuation, yes?

Reply:
> For line continuation, yes?

Dunno. It was there from previous contributions. But I guess yeah